versions.yaml

```
@@ -1,6 +1,6 @@
 supported:
   - channel: '0.53'
-    version: 'v0.53.0-rc2'
+    version: 'v0.53.0-rc4'
```

Any reason to skip rc3?

```suggestion
    version: 'v0.53.0-rc3'
```